CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Changed
```

Comment:
Should we mention the fix for the build issue when using a new Player version in combination with the conviva integration?

Reply:
Ah yes, we should!
Fixed in 258f880